### ISSUE TITLE
FAIL: //tensorflow/core/grappler/clusters:single_machine_test

### ISSUE THREAD

**Issue by lukeiwanski** [lukeiwanski/tensorflow#77]
## System Info

```
  Name:						 Hawaii
  Vendor:					 Advanced Micro Devices, Inc.
  Device OpenCL C version:			 OpenCL C 2.0 
  Driver version:				 1912.5 (VM)
  Profile:					 FULL_PROFILE
  Version:					 OpenCL 2.0 AMD-APP (1912.5)
  Extensions:					 cl_khr_fp64 cl_amd_fp64 cl_khr_global_int32_base_atomics cl_khr_global_int32_extended_atomics cl_khr_local_int32_base_atomics cl_khr_local_int32_extended_atomics cl_khr_int64_base_atomics cl_khr_int64_extended_atomics cl_khr_3d_image_writes cl_khr_byte_addressable_store cl_khr_gl_sharing cl_khr_gl_depth_images cl_ext_atomic_counters_32 cl_amd_device_attribute_query cl_amd_vec3 cl_amd_printf cl_amd_media_ops cl_amd_media_ops2 cl_amd_popcnt cl_khr_image2d_from_buffer cl_khr_spir cl_khr_subgroups cl_khr_gl_event cl_khr_depth_images cl_khr_mipmap_image cl_khr_mipmap_image_writes

```
ComputeCpp 0.2.0

## To reproduce
```
bazel test --config=sycl --local_test_jobs=4 -k --test_lang_filters=cc,py --action_env=LD_PRELOAD=/usr/lib/libOpenCL.so.1 --test_timeout 300,750,1200,3600 //tensorflow/core/grappler/clusters:single_machine_test
```

## Error
```
[ RUN      ] SingleMachineTest.Queue
2017-06-12 15:55:49.124896: I ./tensorflow/core/grappler/clusters/single_machine.cc:269] Starting new session
2017-06-12 15:55:49.138891: I ./tensorflow/core/grappler/clusters/single_machine.cc:257] Cleaning up previous session
2017-06-12 15:55:49.142450: I ./tensorflow/core/grappler/clusters/single_machine.cc:269] Starting new session
2017-06-12 15:55:54.143757: F ./tensorflow/core/grappler/clusters/single_machine_test.cc:90] Non-OK-status: cluster_->Run(item.graph, item.feed, item.fetch, &metadata) status: Deadline exceeded: Failed to run the graph after 5 seconds, aborting
external/bazel_tools/tools/test/test-setup.sh: line 159: 19098 Aborted                 (core dumped) "${TEST_PATH}" "$@"
(END)
```

**Comment by jwlawson** [comment]
For me the failure is in `SingleMachineTest.CostModel`:
```
[ RUN      ] SingleMachineTest.CostModel
2017-06-14 10:43:59.108073: I ./tensorflow/core/grappler/clusters/single_machine.cc:257] Cleaning up previous session
2017-06-14 10:43:59.108701: I ./tensorflow/core/grappler/clusters/single_machine.cc:269] Starting new session
./tensorflow/core/grappler/clusters/single_machine_test.cc:72: Failure
Expected: (8) <= (node.output_info(0).size()), actual: 8 vs 0
[  FAILED  ] SingleMachineTest.CostModel (300 ms)
```
And the `SingleMachineTest.Queue` test passes. Tried both up to date `eigen_mehdi` and `dev/eigen_mehdi` at be13983.

**Comment by lukeiwanski** [comment]
Yes that's the current issue.

**Comment by jwlawson** [comment]
Fixed by #114

**Comment by lukeiwanski** [comment]
Closing
